protobuf/transaction.proto

```diff
@@ -26,68 +26,46 @@ import "protobuf/query.proto";
 
 package grakn.protocol;
 
+service TransactionService {
+
+    // Opens a bi-directional stream representing a stateful transaction, streaming
+    // requests and responses back-and-forth. The first request message must
+    // be {Transaction.Open.Req}. Closing the stream closes the transaction.
+    rpc stream (stream Transaction.Req) returns (stream Transaction.Res);
+}
+
 message Transaction {
 
     message Req {
-        map<string, string> metadata = 1000;
+        string id = 1;
+        map<string, string> metadata = 2;
+        int32 latencyMillis = 3;
         oneof req {
-            Open.Req open_req = 1;
-            Commit.Req commit_req = 2;
-            Rollback.Req rollback_req = 3;
-            Iter.Req iter_req = 4;
-            Query.Req query_req = 5;
-            GetType.Req getType_req = 6;
-            GetThing.Req getThing_req = 7;
-            GetRule.Req getRule_req = 8;
-            PutEntityType.Req putEntityType_req = 9;
-            PutAttributeType.Req putAttributeType_req = 10;
-            PutRelationType.Req putRelationType_req = 11;
-            PutRule.Req putRule_req = 12;
-            ThingMethod.Req thingMethod_req = 13;
-            TypeMethod.Req typeMethod_req = 14;
-            RuleMethod.Req ruleMethod_req = 15;
-            //Explanation.Req explanation_req = 16;
+            Fetch.Req fetch_req = 4;
```

Comment:
`Fetch.Req` is an indirect replacement for the old `Iter.Req` - it must be called in combination with a request ID that is already being processed on the server, and is used to fetch the next batch of answers.

Reply:
This should just be a `bool continue`